src/data_storage/sqlalchemy/models.py

```diff
@@ -0,0 +1,55 @@
+import json
+import logging
+from functools import partial
+
+from pydantic.json import pydantic_encoder
+from sqlalchemy import VARCHAR, Column, ForeignKey, Integer, create_engine
+from sqlalchemy.ext.declarative import declarative_base
+
+from conf import settings
+
+
+def get_connection():
+    # Setup Session and Client
+    logging.info("Getting database connection")
+    conn = create_engine(
+        f"postgresql://{settings.DB_USER}:{settings.DB_PASS}@{settings.DB_HOST}:{settings.DB_PORT}/{settings.DB_NAME}",
+        json_serializer=partial(json.dumps, default=pydantic_encoder),
+    )
+    logging.info("Database connection established")
+    return conn
+
+
+engine = get_connection()
+Base = declarative_base(bind=engine)
+
+
+class Story(Base):
+    __tablename__ = "story"
+
+    id = Column(Integer, primary_key=True)
+    name = Column(VARCHAR(200), nullable=False)
+
+
+class Node(Base):
+    __tablename__ = "node"
+
+    id = Column(Integer, primary_key=True)
+    story = Column(
+        Integer,
+        ForeignKey("story.id", ondelete="CASCADE"),
+        nullable=True,
+    )
+    name = Column(VARCHAR(200), nullable=False, index=True)
```

Comment:
Why don't we use https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.String?